src/Fetchers/KNHBFetcher/KNHBClubFetcher.ts

```diff
@@ -36,14 +36,17 @@ export class KNHBClubFetcher {
     }
 
     /**
-     * Simplify the input string, used to match club names.
-     * @param input The club name.
-     * @protected
+     * Simplify a string for comparison.
+     * @param input The input string.
      */
-    public static simplifyString(input: string) {
+    public static simplifyString(input: string | null): string {
+        if (!input) return "";
+
         return input
             .toLowerCase()
-            .replaceAll(/[^a-z]/g, "");
+            .normalize("NFD")
```

Comment:
Why is this necessary to implement officals?

Reply:
It was solving a bug at some point but should have been removed.